src/cells/model.ts

```diff
@@ -357,7 +357,7 @@ class CodeCellModel extends BaseCellModel implements ICodeCellModel {
   constructor(input: IInputAreaModel, output: IOutputAreaModel) {
     super(input);
     this._output = output;
-    this.input.prompt = 'In [ ]:';
+    this.input.prompt = ' ';
```

Comment:
I'd even say that the space is a view-layer thing.  If the prompt is the empty string, then the view layer displays a space so that things line up correctly.